config/externalname.go

```diff
@@ -2233,10 +2233,10 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	//
 	// Network Firewall Policies can be imported using their ARN
 	// Example: arn:aws:network-firewall:us-west-1:123456789012:firewall-policy/example
-	"aws_networkfirewall_firewall_policy": config.TemplatedStringAsIdentifier("name", "arn:aws:network-firewall:{{ .parameters.region }}:{{ .setup.client_metadata.account_id }}:firewall-policy/{{ .external_name }}"),
+	"aws_networkfirewall_firewall_policy": config.IdentifierFromProvider,
```

Comment:
When we introduce the `spec.forProvider.name` back to the firewall policy resource, we will also need to add it to the manually maintained manifests: 
https://github.com/upbound/provider-aws/blob/1faeb9c0b215155b2c62cf13fc1200dd4dabb94d/examples/networkfirewall/firewallpolicy.yaml#L10